src/packages/@ncigdc/components/Aggregations/TermAggregation.js

```diff
@@ -71,149 +71,148 @@ const TermAggregation = (props: TProps) => {
           ((ctx.query &&
             parseFilterParam((ctx.query || {}).filters, {}).content) ||
           [])
-          .map(filter => ({
-            ...filter,
-            content: {
-              ...filter.content,
-              value: typeof filter.content.value === 'string'
+            .map(filter => ({
+              ...filter,
+              content: {
+                ...filter.content,
+                value: typeof filter.content.value === 'string'
                 ? filter.content.value.toLowerCase()
-                : filter.content.value.map(val => val.toLowerCase())
+                : filter.content.value.map(val => val && `${val}`.toLowerCase()),
```

Comment:
What the type value can be? Is that ok to assume that the value is an array if it's not a string?

Reply:
If the value is neither a string or an array, then we have an API issue.
But you're right, we could add an extra check here. Await amendment.